src/icebergs.F90

```diff
@@ -4109,6 +6405,682 @@ subroutine calve_icebergs(bergs)
 
 end subroutine calve_icebergs
 
+  !> Calve footloose icebergs from parent berg
+subroutine calve_fl_icebergs(bergs,pberg,k,l_b,fl_disp_x,fl_disp_y,berg_from_bits)
+  ! Arguments
+  type(icebergs), pointer :: bergs !< Container for all types and memory
+  type(iceberg), pointer :: pberg !< Parent berg
+  type(real),intent(in) :: k !< Number of child bergs to calve
```

Comment:
Two issues for these two lines:
1. `type(real)` should be `real` (also for `l_b` in the next line)
2. I don't see where `k` is used later. If it is the number of bergs I would have expected it to be an integer, but since it is not used, it both doesn't matter and probably should not be an argument at all?